anime/views.py

```diff
@@ -44,3 +44,19 @@ def get_anime_by_name(name: str):
         return anime_schema.dump(row)
     else:
         abort(404)
+
+
+@animes_bp.route('/api/v1/animes/<int:id>', methods=['Patch', 'DELETE'])
+def anime(id: int):
+    """Route updates attribute(s) of single anime in DB
+
+    :param name: Anime name to update its attribute(s)
+    :type name: str
+    :return: Updated anime record
+    :rtype: AnimeSchema
+    """
+
+    if request.method == 'DELETE':
+        db.session.query(Anime).filter(Anime.anime_id == id).delete()
```

Comment:
You will get an error if the id is not found here because the filter will be empty, and deletion cannot be done on an empty list.

Reply:
I tested the method when an anime doesn't exist in the DB, and return 204 with empty string (success), it seems no errors raised. But I'm working on the method to return error when an anime doesn't exist - makes more sense!

Reply:
changes in commit # 2a35f67 upon this SO https://stackoverflow.com/questions/27158573/how-to-delete-a-record-by-id-in-flask-sqlalchemy. if/elif logic created upon a comment in the solution, which is "the delete() method returns the number of items deleted. So if User does not exist, it returns zero and throws no exception. I tested this behavior on Sqlalchemy v1.4.29 with a SQLite in-memory database".